\PassOptionsToPackage{sort&compress}{natbib}
\documentclass[preprint,3p,times,twocolumn]{elsarticle}
\usepackage{graphicx}
\usepackage{color}
\usepackage{amssymb}
\usepackage{amsmath}
\usepackage{multirow}
\usepackage{rotating,capt-of}
\usepackage[small]{caption}
\usepackage{booktabs}
\usepackage{fixltx2e} % two-column fig/tab float order
\usepackage{float} % for placing figures where i want
\usepackage{afterpage}
\usepackage{soul}  % KR: added as it contains \st command
\usepackage{epsfig, a4wide}
\usepackage{floatflt}

\newcommand{\myparagraph}[1]{
  \paragraph*{\normalfont\itshape #1}\hspace{5pt}}

% strange snos
\definecolor{purple}{RGB}{180,90,200}
\definecolor{dgreen}{RGB}{0,160,0}
\definecolor{turquoise}{RGB}{0,180,140}
\renewcommand\dblfloatpagefraction{0.03}
\renewcommand\topfraction{.95}
\renewcommand\bottomfraction{.95}
\renewcommand\textfraction{.05}
\renewcommand\floatpagefraction{.95}
\renewcommand\dbltopfraction{.95}
\renewcommand\dblfloatpagefraction{.95}
%\newcommand{\PFS}[1]{\begingroup\color{blue}#1\endgroup}
%\newcommand{\DOM}[1]{\begingroup\color{green}#1\endgroup}
\newcommand{\JH}[1]{\begingroup\color{purple}#1\endgroup}
%\newcommand{\KR}[1]{\begingroup\color{magenta}#1\endgroup}
\newcommand{\SB}[1]{\begingroup\color{turquoise}#1\endgroup}
\newcommand{\TODO}[1] {\begingroup\color{red}#1\endgroup}
%\newcommand{\FINAL}[1]{\begingroup\color{dgreen}#1\endgroup}
\newcommand{\ACC}[1]{\emph{\textbf{#1}}}
\newcommand{\s}[1]{\begin{tiny}#1\end{tiny}}
\newcommand{\url}[1]{\texttt{\small #1}}
\newcommand{\maxentscan}{\texttt{MaxEntScan}}

% Sebastian (Diss)
%% abbreviations
\newcommand{\snos}{snoRNAs}
\newcommand{\sno}{snoRNA}
\newcommand{\cd}{box C/D snoRNA}
\newcommand{\haca}{box H/ACA snoRNA}
%% programs
\newcommand{\blast}{\texttt{Blast}}
\newcommand{\snostrip}{\texttt{snoStrip}}
\newcommand{\epope}{\texttt{ePoPE}}
\newcommand{\plexy}{\texttt{PLEXY}}
\newcommand{\snoop}{\texttt{RNAsnoop}}
\newcommand{\fold}{\texttt{RNAfold}}
\newcommand{\alifold}{\texttt{RNAalifold}}
\newcommand{\locarna}{\texttt{LocaRNA}}
%% databases
\newcommand{\ncbi}{\texttt{NCBI}}
\newcommand{\umass}{\texttt{UMass}}



%FUNGI
\newcommand{\afu}{\emph{A.fumigatus}}
\newcommand{\Afu}{\emph{Aspergillus fumigatus}}
\newcommand{\nfi}{\emph{N.fischeri}}
\newcommand{\Nfi}{\emph{Neosartorya fischeri}}
\newcommand{\calb}{\emph{C.albicans}}
\newcommand{\Calb}{\emph{Candida albicans}}
\newcommand{\schizo}{\emph{Schizosaccharomyces}}
\newcommand{\fusarium}{\emph{Fusarium}}
\newcommand{\neurospora}{\emph{Neurospora}}
\newcommand{\trichoderma}{\emph{Trichoderma}}
\newcommand{\botrytis}{\emph{Botrytis}}
\newcommand{\sclerotinia}{\emph{Sclerotinia}}
\newcommand{\aspergillus}{\emph{Aspergillus}}
\newcommand{\penicillium}{\emph{Penicillium}}
\newcommand{\saccharomyces}{\emph{Saccharomyces}}
\newcommand{\dothi}{Dothideomycetes}
\newcommand{\tde}{\emph{T.deformans}}
\newcommand{\Tde}{\emph{Taphrina deformans}}
\newcommand{\Spo}{\emph{Schizosaccharomyces pombe}}
\newcommand{\spo}{\emph{S.pombe}}
\newcommand{\Sja}{\emph{Schizosaccharomyces japonicus}}
\newcommand{\sja}{\emph{S.japonicus}}
\newcommand{\scm}{\emph{S.complicata}}
\newcommand{\Scm}{\emph{Saitoella complicata}}
\newcommand{\Ssl}{\emph{Sclerotinia sclerotiorum}}
\newcommand{\ssl}{\emph{S.sclerotiorum}}
\newcommand{\Clo}{\emph{Chalara longipes}}
\newcommand{\clo}{\emph{C.longipes}}
\newcommand{\Pan}{\emph{Podospora anserina}}
\newcommand{\pan}{\emph{P.anserina}}
\newcommand{\Oma}{\emph{Oidoidendron maius}}
\newcommand{\oma}{\emph{O.maius}}
\newcommand{\Fgr}{\emph{Fusarium graminearum}}
\newcommand{\fgr}{\emph{F.graminearum}}
\newcommand{\fve}{\emph{F.verticillioides}}
\newcommand{\Fve}{\emph{Fusarium verticillioides}}
\newcommand{\Fox}{\emph{Fusarium oxysporum}}
\newcommand{\fox}{\emph{F.oxysporum}}
\newcommand{\alternaria}{\emph{Alternaria}}
\newcommand{\Abr}{\emph{Alternaria brassicicola}}
\newcommand{\abr}{\emph{A.brassicicola}}
\newcommand{\mycosphaerella}{\emph{Mycosphaerella}}
\newcommand{\Mpi}{\emph{Mycosphaerella pini}}
\newcommand{\mpi}{\emph{M.pini}}
\newcommand{\pex}{\emph{Penicillium expansum}}
\newcommand{\pcy}{\emph{Penicillium chrysogenum}}
\newcommand{\Rha}{\emph{Rhodotorula hasegawae}}
\newcommand{\rha}{\emph{R.hasegawae}}
\newcommand{\Rda}{\emph{Rhodosporidium dacryoidum}}
\newcommand{\rda}{\emph{R.dacryoidum}}
\newcommand{\Rgr}{\emph{Rhodotorula graminis}}
\newcommand{\rgr}{\emph{R.graminis}}
\newcommand{\Rmi}{\emph{Rhodotorula minuta}}
\newcommand{\rmi}{\emph{R.minuta}}
\newcommand{\Sli}{\emph{Sporobolomyces linderae}}
\newcommand{\sli}{\emph{S.linderae}}
\newcommand{\Cne}{\emph{Cryptococcus neoformans}}
\newcommand{\cne}{\emph{C.neoformans}}
\newcommand{\Ncr}{\emph{Neurospora crassa}}
\newcommand{\ncr}{\emph{N.crassa}}
\newcommand{\Nte}{\emph{Neurospora tetrasperma}}
\newcommand{\nte}{\emph{N.tetrasperma}}
\newcommand{\Andi}{\emph{Aspergillus nidulans}}
\newcommand{\andi}{\emph{A.nidulans}}
\newcommand{\Sce}{\emph{Saccharomyces cerevisiae}}
\newcommand{\sce}{\emph{S.cerevisiae}}
\newcommand{\Spa}{\emph{Saccharomyces paradoxus}}
\newcommand{\spa}{\emph{S.paradoxus}}
\newcommand{\Smi}{\emph{Saccharomyces mikatae}}
\newcommand{\smi}{\emph{S.mikatae}}
\newcommand{\Kpa}{\emph{Komagataella pastoris}}
\newcommand{\kpa}{\emph{K.pastoris}}
\newcommand{\Cpr}{\emph{Cryphonectria parasitica}}
\newcommand{\cpr}{\emph{C.parasitica}}
\newcommand{\Glo}{\emph{Glarea lozoyensis}}
\newcommand{\glo}{\emph{G.lozoyensis}}
\newcommand{\Yli}{\emph{Yarrowia lipolytica}}
\newcommand{\yli}{\emph{Y.lipolytica}}
\newcommand{\Ecu}{\emph{Encephalitozoon cuniculi}}
\newcommand{\ecu}{\emph{E.cuniculi}}
\newcommand{\Ror}{\emph{Rhizopus oryzae}}
\newcommand{\ror}{\emph{R.oryzae}}
\newcommand{\Tre}{\emph{Trichoderma reesei}}
\newcommand{\tre}{\emph{T.reesei}}
\newcommand{\Npa}{\emph{Nematocida parisii}}
\newcommand{\npa}{\emph{N.parisii}}
\newcommand{\Eae}{\emph{Edhazardia aedis}}
\newcommand{\eae}{\emph{E.aedis}}
\newcommand{\Vco}{\emph{Vittaforma corneae}}
\newcommand{\vco}{\emph{V.corneae}}
\newcommand{\Mbi}{\emph{Metschnikowia bicuspidata}}
\newcommand{\mbi}{\emph{M.bicuspidata}}
\newcommand{\Mva}{\emph{Meliniomyces variabilis}}
\newcommand{\mva}{\emph{M.variabilis}}
\newcommand{\Bci}{\emph{Botrytis cinerea}}
\newcommand{\bci}{\emph{B.cinerea}}
\newcommand{\Val}{\emph{Verticillium alfalfae}}
\newcommand{\val}{\emph{V.alfalfae}}
\newcommand{\Vda}{\emph{Verticillium dahliae}}
\newcommand{\vda}{\emph{V.dahliae}}
\newcommand{\Sal}{\emph{Sodiomyces alkalinus}}
\newcommand{\sal}{\emph{S.alkalinus}}
\newcommand{\Aac}{\emph{Acremonium alcalophilum}}
\newcommand{\aac}{\emph{A.~alcalophilum}}
\newcommand{\Nha}{\emph{Nectria haematococca}}
\newcommand{\nha}{\emph{N.haematococca}}
\newcommand{\Mth}{\emph{Myceliophthora thermophila}}
\newcommand{\mth}{\emph{M.thermophila}}
\newcommand{\Cgl}{\emph{Chaetomium globosum}}
\newcommand{\cgl}{\emph{C.globosum}}
\newcommand{\Tar}{\emph{Thielavia arenaria}}
\newcommand{\tar}{\emph{T.arenaria}}
\newcommand{\Tte}{\emph{Thielavia terrestris}}
\newcommand{\tte}{\emph{T.terrestris}}
\newcommand{\Ztr}{\emph{Zymoseptoria tritici}}
\newcommand{\ztr}{\emph{Z.tritici}}
\newcommand{\Tvi}{\emph{Trichoderma virens}}
\newcommand{\tvi}{\emph{T.virens}}
\newcommand{\Bco}{\emph{Baudoinia compniacensis}}
\newcommand{\bco}{\emph{B.compniacensis}}
\newcommand{\Kla}{\emph{Kluyveromyces lactis}}
\newcommand{\kla}{\emph{K.lactis}}
\newcommand{\Cga}{\emph{Candida glabrata}}
\newcommand{\cga}{\emph{C.glabrata}}
\newcommand{\Pme}{\emph{Pichia membranifaciens}}
\newcommand{\pme}{\emph{P.membranifaciens}}
\newcommand{\Ppl}{\emph{Postia placenta}}
\newcommand{\ppl}{\emph{P.placenta}}
\newcommand{\Nfu}{\emph{Nadsonia fulvescens}}
\newcommand{\nfu}{\emph{N.fulvescens}}
\newcommand{\Asp}{\emph{Atractiellales sp}}
\newcommand{\asp}{\emph{A.sp}}
\newcommand{\Bma}{\emph{Bipolaris maydis}}
\newcommand{\bma}{\emph{B.maydis}}
\newcommand{\Ptt}{\emph{Pyrenophora tritici-repentis}}
\newcommand{\ptt}{\emph{P.tritici-repentis}}
\newcommand{\Ana}{\emph{Aureobasidium namibiae}}
\newcommand{\ana}{\emph{A.namibiae}}
\newcommand{\Cbb}{\emph{Cucurberita berberidis}}
\newcommand{\Pno}{\emph{Parastagonospora nodorum}}
\newcommand{\wse}{\emph{W.sebi}}
\newcommand{\Wse}{\emph{Wallemia sebi}}
\newcommand{\opi}{\emph{O.piceae}}
\newcommand{\Opi}{\emph{Ophiostoma piceae}}
\newcommand{\cdu}{\emph{C.dubliniensis}}
\newcommand{\Cdu}{\emph{Candida dubliniensis}}
\newcommand{\ctr}{\emph{C.tropicalis}}
\newcommand{\Ctr}{\emph{Candida tropicalis}}


%Trennung verhindern
\hyphenation{snoRNAs}
\hyphenation{snoRNA}
\hyphenation{microRNA}
\hyphenation{microRNAs}
\hyphenation{miRNA}
\hyphenation{miRNAs}
\hyphenation{lncRNA}
\hyphenation{lncRNAs}
\hyphenation{lincRNA}
\hyphenation{lincRNAs}


\journal{Preprint}

\DeclareCaptionLabelFormat{simplesupp}{#1~S#2} % new caption format with Sxx

\begin{document}

\begin{frontmatter}

\title{The Fungal snoRNAome}

\author[LEI]{Sebastian Canzler\corref{cor1}}
\ead{sebastian@bioinf.uni-leipzig.de}
%\author[LEI]{Stephanie Kehr}
%\ead{steffi@bioinf.uni-leipzig.de}
\author[LEI,IZBI,MPI,IZI,RTH,TBI,SFI]{Peter F.\ Stadler}
\ead{studla@bioinf.uni-leipzig.de}
\author[UFZ]{Jana Hertel}
\ead{jana.hertel@ufz.de}  % KR: new E-Mail address

\address[LEI]{Bioinformatics Group, Department of Computer Science,
              University Leipzig,
              H{\"a}rtelstra{\ss}e 16-18, D-04107 Leipzig, Germany
             }
\address[IZBI]{Interdisciplinary Center for Bioinformatics,
              University Leipzig,
              H{\"a}rtelstra{\ss}e 16-18, D-04107 Leipzig, Germany
             }
\address[UFZ]{Young Investigators Group Bioinformatics and Transcriptomics, Department Proteomics, 
              Helmholtz Centre for Environmental Research -- UFZ, 
              Permoserstra{\ss}e 15,
              D-04318 Leipzig, Germany
             }
\address[IZI]{Department of Diagnostics, Fraunhofer Institute for Cell Therapy and Immunology -- IZI,
              Perlickstra{\ss}e 1, 
              D-04103 Leipzig, Germany
             }
\address[MPI]{Max Planck Institute for Mathematics in the Sciences,
              Inselstra{\ss}e 22, D-04103 Leipzig, Germany
             }
\address[TBI]{Department of Theoretical Chemistry,
              University of Vienna,
              W{\"a}hringerstra{\ss}e 17, A-1090 Wien, Austria
             }
\address[RTH]{Center for non-coding RNA in Technology and Health,
              University of Copenhagen, Gr{\o}nneg{\aa}rdsvej 3, 
              DK-1870 Frederiksberg C, Denmark
             }
\address[SFI]{Santa Fe Institute, 1399 Hyde Park Rd., Santa Fe, NM 87501,
              USA\\[1em]
             }

%\cortext[jfa]{Joint first authors}
\cortext[cor1]{Corresponding author}

% --------------------------------------------------------------------------- %


\begin{abstract}
%  Large-scale RNA sequencing has revealed a large number of long mRNA-like
%  transcripts (lncRNAs) that do not code for proteins. The evolutionary
%  history of these lncRNAs has been notoriously hard to study
%  systematically due to their low level of sequence conservation that
%  precludes comprehensive homology-based surveys and makes them nearly
%  impossible to align. An increasing number of special cases, however, has
%  been shown to be at least as old as the vertebrate lineage. Here we use
%  the conservation of splice sites to trace the evolution of lncRNAs. We
%  show that more than $85$\,\% of the human GENCODE lncRNAs were
%  already present at the divergence of placental mammals and many hundreds
%  of these RNAs date back even further. Nevertheless, we observe a fast
%  turnover of intron/exon structures. We conclude that lncRNA genes are
%  evolutionary ancient components of vertebrate genomes that show an
%  unexpected and unprecedented evolutionary plasticity. We offer a public
%  web service (\url{http://splicemap.bioinf.uni-leipzig.de}) that allows to
%  retrieve sets of orthologous splice sites and to produce overview maps of
%  evolutionarily conserved splice sites for visualization and further
%  analysis. 
  
  An electronic supplement containing the data sets used and produced in
  this study is available at
  \url{http://www.bioinf.uni-leipzig.de/publications/supplements/17-001}. 
\end{abstract}

\begin{keyword}
  small nucleolar RNAs, snoRNA, fungi, evolution, target switch, snoRNA target, 	
  conservation	
\end{keyword}

\end{frontmatter}

% --------------------------------------------------------------------------- %

\section{Introduction}

Small nucleolar RNAs (snoRNAs) are non-protein-coding RNAs (ncRNAs) that
guide the chemical modification of single nucleotides in other RNA
molecules.  Localized in the nucleolus of eukaryotic (and some archaean)
cells, they associate with at least four proteins to form the small
nucleolar ribonucleoprotein (snoRNP) complex \cite{****}.  The target RNA
molecule is held in the correct position by base pairing to short unpaired
region(s) within the snoRNA usally referred to as the antisense elements
(ASE). The base pairing completely specifies the target nucleotide. Known
modification targets mostly are located in ribosomal RNAs (rRNAs) and small
nuclear RNAs (snRNAs), although some snoRNAs \TODO{* other RNAs* } are also
among the targets \cite{Clouet_d'Orval:2001, Dennis:2001, Uliel:2004,
  Cavaillé:2000, Kishore:2006, Kiss:2002, Matera:2007, Bratkovi:2011}.

There are two distinct classes of snoRNAs: box C/D and box H/ACA snoRNAs.
They are distinguished by their secondary structure, sequence features, and
the modification that they guide \cite{****}.  Box C/D snoRNAs form a
stem-loop structure with a rather long loop which is stabilized by the
associated proteins and guide the 3'OH-methylation of nucleotides.  Box
H/ACA snoRNAs are longer, fold into a thermodynamically more stable double
stem-loop structure, and guide pseudouridylation of uracil residues in the
target RNA.  In addition to these two classes, there are chimeric snoRNAs
that share features of both classes, are much longer and/or are described
to have different functions \cite{Darzacq:2002}. Similar to other small
ncRNAs, snoRNAs require both specific secondary structures and
characteristic sequence motifs to perform their function. These features
are therefore preserved by evolution and are clearly recognizable by
comparative methods \cite{****}. While sequence motifs involved in protein
binding are common to all members of each of the the two classes, the ASEs
are conserved only among members snoRNA families with the same
target. Overall, therefore, snoRNA sequences evolve rapidly, making them
hard to identify by purely sequence-base methods such as \texttt{blast}
\cite{Altschul:1990}.

To overcome this limitation, we introduced a computational annotation
pipeline \snostrip \cite{Bartschat:2014} that is specifically designed to
track all specific characteristics of snoRNAs. Here we use this approach to
analyse a large set of fungal species with genomes that are available in
decent quality for their snoRNA abundance.  We started with experimentally
verified snoRNAs in five fungi.  Further, we studied the evolutionary
conservation of those snoRNAs and the co-evolution of snoRNAs and their
targets. We provide a comprehensive set of fungal snoRNAs, their detailed
description with respect to genomic location, box motifs and position,
potential/confirmed target information (including observed target
switches), family assignment and a suggestion of the evolutionary history
of individual snoRNA families.  All data can be viewed in and downloaded
from our electronic supplement.

% --------------------------------------------------------------------------- %

\section{Materials and Methods}

\subsection{Genome and snoRNA Data} 

Genome sequences from 147 fungal species were downloaded from Ensembl
Genomes \cite{Kersey:2016}, JGI \cite{Nordberg:2014}, Broad Institute
(Fungal Genome Initiative) , and Candida Genome Database
\cite{Skrzypek:2017}.  An \ncbi-based taxonomic tree displaying the
relationship, genome source, and version for all fungal organisms in
this evolutionary survey is shown in the supplementary Figures S1.
%A simplified version of that showing the important fungal lineages is provided in Figure \ref{fig:simplified_tree}. 

%\begin{figure}[b]
%  \centering
%  \includegraphics[width=0.9\textwidth]{simplified_ncbi_tree.eps}
%  \caption[Taxonomic tree of major fungal lineages.]{Simplified \ncbi-derived taxonomic tree displaying major fungal lineages.}
%  \label{fig:simplified_tree}
%\end{figure}

For 63 out of the 147 species, snoRNA sequences were already retrieved in 
%at the \snostrip\
%webserver\footnote{\url{http://snostrip.bioinf.uni-leipzig.de/}}.
%This data is based on 
a previous \snostrip\ study \cite{Bartschat:2014}
that started with experimentally detected snoRNAs extracted from five
surveys for \Ncr\ \citep{Liu:2009}, \Afu\ \citep{Joechl:2008}, \Calb\
\citep{Mitrovich:2010}, \Sce\ \citep{Piekna-Przybylska:2007}, and
\Spo\ \citep{Li:2005}. An overview of the exerimentally verified \snos\ and the
corresponding publications can be seen in supplementary Table S2.  An
entire mapping of the species-specific \sno\ names as they were used in
the original publications and the internal \snostrip\ names is shown
in supplementary Tables S3. 
%\JH{Is the following a result from the
%previous study \cite{Bartschat:2014} or from this one? This does not
%feel like a methods paragraph.} Note that annotated \afu\ \cd\
%AM921943 is treated as a \haca.  It shows two separated, perfect
%hairpins and comprises convincing box motifs while it clearly lacks
%characteristics of \cd.  In addition, \afu\ \cd s AM921919 and
%AM921934 are treated as a single \sno. Both sequences map to the same
%genomic location. Though, AM921934 comprises three point-mutations
%compared to AM921919. 
%Budding yeast \sno\ sequences have been
%downloaded from the
%\texttt{UMass}-database\footnote{http://people.biochem.umass.edu/fournierlab/snornadb/main.php}.
%All other \sno s have been taken from the corresponding
%publication. 
In total, our starting set of snoRNAs comprises 3564
snoRNA sequences assigned to 123 snoRNA families in the 63 species including 231 \sno\ genes taken from the five publications.

\subsection{Homology search}

\snostrip \citep{Bartschat:2014} - our snoRNA annotation pipeline - was
applied to the set of collected snoRNAs and the 147 fungal species in
an iterative manner. Starting with Pezizomycotina, followed by
Saccharomycotina, and other lineages towards the root of the
phylogenetic tree. Each time new (plausible) homologous snoRNAs were
detected, the procedure was repeated to decrease the number of false
negatives until no novel homologs were found anymore.

\subsection{Data curation}

The \snostrip\ retrieved homologs were curated regarding the
automatically identified box motifs, correct molecule lengths, and the
overall fit of each \sno\ sequence in its respective family.  To
identify falsely annotated box motifs, the conservation of all
automatically selected boxes was checked by a comparison of the start
positions within the \sno\ family alignment. Motifs that have \SB{unconserved} start
positions were re-adjusted to fit the \sno\ family specific box pattern and and box position.
\JH{what is the criteria for beeing
  conserved or not?} 
% If this was not feasible due to non-canonical box motifs at the
% supposed positions, either the box annotation was deleted (prime
% boxes) or the whole \sno\ candidate was rejected.  since disrupted
% box motifs imply non-functional mature \sno s.
Sequences that were too large or too short were automatically adjusted according to box
motifs, sequence and structure conservation. 

\subsection{Box motifs, sequence and structure}

Box motifs were generated from all \snostrip-derived \sno\ candidates
and compared to canonical box motifs of yeast and vertebrate snoRNAs.
Sequence lengths and distances between all box motifs were collected
and compared. Secondary structure prediction was done using the \fold\
and \alifold\ programs from the Vienna RNA
Package\cite{Hofacker:1994}.

\subsection{Phylogenetic analysis}

To follow the evolution of the snoRNA families along the phylogenetic
tree we used the software \epope\cite{Hertel:2015}.  It implements a
variant of Sankoff’s parsimony algorithm using the Dollo variant that
excludes the loss and re-gain of a gene family along the same lineage
during evolution.  Innovation and deletion$/$loss$/$divergence events
are deduced and mapped to the branches of the phylogenetic tree.  The
\epope\ results are combined for \emph{all} \sno\ families using the
\texttt{ePoPE\_summarize.pl} tool that comes with the \epope\
distribution.

\subsection{Target prediction and analysis}

Target prediction is part of the \snostrip\ pipeline. There, the
computational tools \plexy\ and \snoop\ are employed to predict
targets for \cd s and \haca s, respectively.  SnoRNAs are investigated
for single or double guide potential based on these predictions and/or
confirmed target interactions. SnoRNAs that remain without target
association are considered orphan. SnoRNAs that are assigned to the
same family but show variance in their associated target are
investigated manually for a potential target switch.


\subsection{Lineage specific conservation of target interactions}
To study the conservation of interactions, the targets for each
individual snoRNA sequence are initially predicted and subsequently
their conservation in other species is evaluated. To formally
investigate the conservation, the Interaction Conservation Index (ICI)
was developed by \cite{Kehr:2014}. Briefly, the conservation of the
modification and the conservation in a specific snoRNA family are
calculated as follows:

\begin{equation}
  \begin{split}
    ICI_{mod}\:(t,s) & = \frac{1}{|O(s)|} * \left( \sum_{k\in O(t,s)} \frac{\varepsilon(t,s,k)}{\bar\varepsilon(t,k)} \right) 
  \end{split}
\end{equation}

\begin{equation}
  \begin{split}
    ICI_{sno}\:(t,s) & = \frac{1}{|O(s)|} * \left( \sum_{k\in O(t,s)} \frac{\varepsilon(t,s,k)}{\hat\varepsilon(s,k)} \right)
  \end{split}
\end{equation}

Where
%\begin{equation}
$ \varepsilon(t,s,k) = \min_{x\in X(t,s,k)} E_{\textrm{mfe}}[x,y_{t,k}] $
%\end{equation}
is scored on family level searching for the best interaction,
i.e. with the lowest minimum free energy, between a snoRNA $x$ of
family $s$ and the target $t$ in species $k$.

Averaging over all predictions of target $t$ in
species $k$
\begin{equation}
\bar\varepsilon(t,k)=\sum_{s\in
  S(t,k)}\varepsilon(t,s,k)/|S(t,k)|
\end{equation}

and averaging over all targets $t$ of snoRNA $s$ in species $k$
\begin{equation}
\hat\varepsilon(s,k)=\sum_{t\in
  T(s,k)}\varepsilon(t,s,k)/|T(s,k)|
\end{equation}

allows the calculation of normalized parameters. These are then
summarized over all species where a prediction of $t$ is found for
snoRNA family $s$ in species $k$ ($k\in O(t,s) $) and normalized over
all species where snoRNA family $s$ is present ($|O(s)|$).


This approach is especially suitable for modification sites that are
present in a large set of analyzed organisms. In cases where a
potential target appears to be lineage specific, the ICI score will
drop to rather low values due to the normalization score $1/O(s)$ that
represents all organisms sharing a homologous \sno\ of family $s$.

To appropriately investigate alternative or additional targets that
merely appear in a particular subset of organisms, the ICI score
calculation has to be adapted to take the particular phylogenetic
distribution of a target interaction into account. Therefore, the
normalization is restricted to the smallest phylogenetic or taxonomic
subtree that harbours all organisms that share prediction of target
$t$ in snoRNA family $s$.  Assume the overall taxonomic tree is
represented by a tree $T=(V,E)$ with root $\gamma$. The minimal
subtree $U_\tau = (V_\tau, E_\tau)$ with root $\tau$ shares the node
set
$V_\tau = \{\ v\ |\ \forall (v,u)\ ,\ u \in V_\tau\ :\ LCA_T(v,u) \in
V_\tau\ \}$ where $LCA_T(v,u)$ is the lowest common ancestor in tree
$T$ of both nodes $v$ and $u$. More precisely, the $LCA$ is the lowest
node, i.e., the farthest node from the root, that has both $v$ and $u$
as descendants.  Hence, the ICI scores in a particular subtree rooted
at $\tau$ can be calculated as follows:
\begin{equation}
  \begin{split}
    ICI_{mod,\tau}\:(t,s) & = \frac{1}{|O_\tau(s)|} * \left( \sum_{k\in O_\tau(t,s)} \frac{\varepsilon(t,s,k)}{\bar\varepsilon(t,k)} \right)
  \end{split}
\end{equation}

\begin{equation}
  \begin{split}
    ICI_{sno,\tau}\:(t,s) & = \frac{1}{|O_\tau(s)|} * \left( \sum_{k\in O_\tau(t,s)} \frac{\varepsilon(t,s,k)}{\hat\varepsilon(s,k)} \right)
  \end{split}
\end{equation}
where
$O_\tau(s) = \{\ k\ |\ \exists t\ :\ X(t,s,k) \ne \emptyset\ \&\ v_k
\in V_\tau\ \}$ denotes the set of organisms that are contained in the
subtree $\tau$ and share at least one snoRNA of family $s$. $v_k$ is
the leaf that denotes organism $k$.

\vspace{5mm}
\par
In this work, \sno\ families are mostly denoted with their internal
\snostrip\ name. Original names are given in parentheses in cases
where previously annotated \snos\ are present, e.g., the internal
\snostrip\ name CD\_22 maps the experimentally detected \sce\ sequence
snR62. A complete mapping of \snostrip\ derived \sno\ names with their
species specific names is provided in the supplement.  A similar
notion is chosen when target positions are described. In most cases,
the alignment position is given and the sequence specific position of
selected organisms are provided in parentheses.

% --------------------------------------------------------------------------- %

\section{Results}
\input{results}
% --------------------------------------------------------------------------- %

\section{Discussion}
\input{discussion}

% --------------------------------------------------------------------------- %

\section*{Acknowledgments}

<<<<<<< HEAD
This work was funded in part by the European Union FP-7 project QUANTOMICS
(no.\ 222664), the MMML-seq project of the International Cancer Genome
Consortium (ICGC) funded by German Federal Ministry of Education and
Research, the CRC 1052 ``Obesity'' \TODO{ and ...}. LIFE -- Leipzig
Research Center for Civilization Diseases is funded by the State of Saxony
and the European Union.
=======
This work was funded in part by the European Union FP-7 project
QUANTOMICS (no.\ 222664). MMML-seq project of the International Cancer
Genome Consortium (ICGC) funded by German Federal Ministry of
Education and Research.  LIFE -- Leipzig Research Center for
Civilization Diseases is funded by the State of Saxony and the
European Union.
>>>>>>> 2e043458cf9138c4628e91cb6a3a42f90e68c792

\bibliographystyle{m}
\bibliography{Fungi_snoRNAome}


%\begin{appendix}
%% use caption format Sxx 
%\captionsetup{labelformat=simplesupp}
%\setcounter{table}{0}
%\setcounter{figure}{0}

% --------------------------------------------------------------------------- %

%\section*{SUPPLEMENT}
%
%\section{Supplementary Figures}
%
%\begin{figure}
%  \centering
%%  \includegraphics[width=\textwidth]{organisms.sorted.upper.half.eps}
%  \caption[Taxonomic Tree of Analyzed Fungi - Part 1]{NCBI-based
%    taxonomic tree of analyzed fungal organisms. NCBI identifier,
%    genome source, and genome version are given after the organism
%    name. All species of the Pezizomycotina lineage are shown in the
%    second part of the figure.}
%  \label{fig:complete_taxonomic_tree_upper_half}
%\end{figure}
%
%\begin{figure}
%  \centering
%%  \includegraphics[width=\textwidth]{organisms.sorted.lower.half.eps}
%  \caption[Taxonomic Tree of Analyzed Fungi - Part 2]{Taxonomic tree
%    of the analyzed Pezizomycotina species. Further explanations as above.}
%  \label{fig:complete_taxonomic_tree_lower_half}
%\end{figure}
%
%\section{Supplementary Tables}
%
%\begin{table}[t!]
%  \caption[Data sources for the fungal snoRNA set.]{\small Summary of
%    publications which provided experimentally detected \snos\
%    that were taken as query sequences in the \emph{\snostrip} pipeline.
%    Please note that the original publication by \citet{Joechl:2008}
%    exclusively contains \cd s. Details see text.
%  }
%  \centering
%  \scriptsize
%%  \begin{rndtableWhite}{|c|l|c|c|l|}
%%\header{}&\header{organism}&\header{\cd}&\header{\haca}&\header{publication}\\
%%\hline
%%\header{}&\emph{N.crassa}&55&20&\citet{Liu:2009}\\
%%\header{}&\emph{A.fumigatus}&25&1&\citet{Joechl:2008}\\
%%\header{}&\emph{C.albicans}&40&-&\citet{Mitrovich:2010}\\
%%\raisebox{3ex}[-3ex]{\header{\rotatebox{90}{Fungi}}}&\emph{S.cerevisiae}&43&29&\citet{Piekna-Przybylska:2007}\\
%%\header{}&\emph{S.pombe}&-&20&\citet{Li:2005}\\
%%\hline
%%\header{}&\hsa&269&108&\citet{Lestrade:2006}\\
%%\header{}&\gga&128&69&\citet{Shao:2009}\\
%%\raisebox{1ex}[-1ex]{\header{\rotatebox{90}{Metazoa}}}&\oan&141&66&\citet{Schmitz:2008}\\
%%\end{rndtableWhite}
%\label{tab:original_snornas}
%\end{table}
%
%\begin{table}[h]
%\vspace{-0.05cm}
%\caption[Mapping of experimentally detected \haca s.]{\small Mapping of
%  experimentally detected \haca s. $^*$ The previously as \cd\
%  classified sequence AM921943 in \Afu\ \citep{Joechl:2008} was mapped to the \haca\
%  family HACA\_21.}
%\centering
%\scriptsize
%%\begin{rndtableWhite}{|c|c|c|c||c|c|c|c|}
%%\header{\snostrip\ name}&\header{\sce}&\header{\spo}&\header{\ncr}&\header{\snostrip\ name}&\header{\sce}&\header{\spo}&\header{\ncr}\\
%%\input{tables/haca_snoRNA_mapping_part1.tex}
%%\end{rndtableWhite}
%\label{tab:haca_snoRNA_mapping}
%\end{table}
%
%
%\begin{table}
%\caption[Mapping of experimentally detected \cd s.]{\small Mapping of
%  experimentally detected \cd s.}
%\centering
%\scriptsize
%%\begin{rndtableWhite}{|c|c|c|c|c|c|}
%%\header{\snostrip\ name}&\header{\sce}&\header{\spo}&\header{\calb}&\header{\afu}&\header{\ncr}\\
%%%\input{tables/cd_snoRNA_mapping_part1.tex}
%%\end{rndtableWhite}
%\label{tab:cd_snoRNA_mapping}
%\end{table}
%
%\addtocounter{table}{-1}
%
%\begin{table}
%\caption[]{\small (Continued.)}
%\centering
%\scriptsize
%%\begin{rndtableWhite}{|c|c|c|c|c|c|}
%%\header{\snostrip\ name}&\header{\sce}&\header{\spo}&\header{\calb}&\header{\afu}&\header{\ncr}\\
%%%\input{tables/cd_snoRNA_mapping_part2.tex}
%%\end{rndtableWhite}
%\end{table}
%
%\section{Supplementary Methods} 
%
%
%\section{Conservation of Protein-Coding Splice Sites}
%
%
%\end{appendix}

\end{document}


